src/handlers/shared/start.ts

```diff
@@ -16,7 +16,14 @@ export async function start(
   teammates: string[]
 ): Promise<Result> {
   const { logger, config } = context;
-  const { taskStaleTimeoutDuration } = config;
+  const { taskStaleTimeoutDuration, requiredLabelsToStart } = config;
+
+  const issueLabels = issue.labels.map((label) => label.name);
+
+  if (requiredLabelsToStart.length && !requiredLabelsToStart.some((label) => issueLabels.includes(label))) {
+    // The "Priority" label must reflect a business priority, not a development one.
+    throw new Error("This task does not reflect a business priority at the moment and cannot be started. This will be reassessed in the coming weeks.");
```

Comment:
By doing so, no message is posted only a log is present (and the log also is actually broken because when the error is a simple error the logger doesn't display it).
```suggestion
    throw logger.error("This task does not reflect a business priority at the moment and cannot be started. This will be reassessed in the coming weeks.");
```